### PR TITLE
Updates README.md

### PR DESCRIPTION
**Pull request by idoroshev**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,39 @@
...
+  "email": null,
+  "userSettings": {
+    "gender": "FEMALE",
+    "birthDate": 944956800000,
+    "weight": 54,
+    "height": 170,
+    "physicalActivity": 1.4,
+    "targetWeight": 52
+  },
+  "password": "��:{X�R�L�u`:�|�"
```
Comment (daksenik) [inline README.md]: Very beautiful. Could you please specify something readable?

File: README.md [hunk 2]
```diff
@@ -1 +1,39 @@
-# Server
\ No newline at end of file
+# Server
+
+## Access
+You can get access to heat backend by url: [https://heat-backend.herokuapp.com](https://heat-backend.herokuapp.com/).
+
+## Methods
+- */registration*
```
Comment (daksenik) [inline README.md]: Plesae, specify the method type (GET/POST/etc.).

**Review by daksenik — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (idoroshev) [reply README.md]: Done.

File: README.md (hunk 2 above)
Comment (idoroshev) [reply README.md]: Done.

**Review by daksenik — APPROVED** [review]

**Pull request merged**